cli/src/vote.rs

```diff
@@ -302,17 +302,7 @@ pub fn process_vote_authorize(
 }
 
 fn get_epoch_schedule(rpc_client: &RpcClient) -> Result<EpochSchedule, Box<dyn std::error::Error>> {
-    let epoch_schedule_account = rpc_client.get_account(&sysvar::epoch_schedule::id())?;
-
-    if epoch_schedule_account.owner != sysvar::id() {
-        return Err(CliError::RpcRequestError(format!(
-            "{:?} is not an epoch_schedule account",
-            sysvar::epoch_schedule::id()
-        ))
-        .into());
-    }
-
-    let epoch_schedule = EpochSchedule::deserialize(&epoch_schedule_account)?;
+    let epoch_schedule = rpc_client.get_epoch_schedule()?;
 
     Ok(epoch_schedule)
```

Comment:
@CriesofCarrots does this look ok? (following mvines suggestion to consolidate w/ the rpc)

Reply:
I think we can replace the `get_epoch_schedule(rpc_client: &RpcClient)` method entirely, and just use `rpc_client.get_epoch_schedule()?` in process_show_vote_account and process_uptime